src/styles/card.scss

```
@@ -10,9 +10,17 @@
   flex-direction: column;
 
   &__photo {
-    width: 160px;
-    height: 134px;
-    margin: 30px 19px;
+    width: 160px + 8px;
+    height: 134px + 28px;
+    margin: 16px auto;
+    border: 3px solid transparent;
+    padding: 11px 1px;
```

Сортуй і розділяй за правилами! У тебе є лінк з правилами що скидали в групі?
```suggestion
    height: 134px + 28px;
    margin: 16px auto;
    padding: 11px 1px;
    
    border: 3px solid transparent;
```

---

поправила в усіх scss файлах сортування, надалі буду вже правильно писати!